tests/language-feature/types/opaque/return-opaque-type-in-struct.slang

```diff
@@ -4,7 +4,7 @@
 // aggregate type that includes an opaque type
 
 //TEST(compute):COMPARE_COMPUTE:
-//DISABLE_TEST(compute):COMPARE_COMPUTE:-slang -shaderobj -mtl
+//TEST(compute):COMPARE_COMPUTE:-slang -shaderobj -mtl
```

Comment:
This test appears to be failing,
```
FAILED test: 'tests/language-feature/types/opaque/return-opaque-type-in-struct.slang.1 (mtl)' 
EXPECTED{{{
12
23
44
85
}}}
ACTUAL{{{
0
0
0
0
}}}
```

Reply:
This PR is still being worked on, I thought it was working, but GFX CI was failing due to an infinite loop I did not test for.